c7n/resources/secretsmanager.py

```diff
@@ -186,6 +186,59 @@ def process(self, resources, event=None):
         return matched
 
 
+@SecretsManager.filter_registry.register('secret-current-version')
```

Comment:
hmm one nit before I merge - possibly worth simplifying to `current-version` since we already know it's a secret?

Reply:
Thanks, makes sense to me. Addressed